project1/styles.css

```diff
@@ -0,0 +1,44 @@
+h1,
+h2,
+h3 {
+    text-align: center;
+    background-color: #eb3f33;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-bottom: 12px;
+}
+
+h1 {
+    font-family: 'Playfair Display', serif;
+}
+
+h2 {
+    font-size: 15px;
+}
+
+h3 {
+    font-family: 'Titillium Web', sans-serif;
+    font-weight: 400px;
```

Comment:
`font-weight` doesn't take units, so this rule is invalid and should likely be:
`font-weight: 400;`